core/parameters/src/config_store.rs

```diff
@@ -196,7 +196,9 @@ impl RuntimeConfigStore {
 
                 // Get the original congestion control config. The nayduck tests
                 // are tuned to this config.
-                let source_protocol_version = ProtocolFeature::CongestionControl.protocol_version();
+                #[allow(deprecated)]
```

Comment:
Remove case for `CONGESTION_CONTROL_TEST` and delete variable `CONGESTION_CONTROL_TEST`